aws-api/src/main/java/com/amplifyframework/api/aws/ModelPagination.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.api.aws;
+
+import androidx.annotation.Nullable;
+
+/**
+ * TODO.
+ */
+public final class ModelPagination {
+
+    private static final int DEFAULT_LIMIT = 1000;
+
+    private int limit;
+    private final String nextToken;
+
+    private ModelPagination(int limit, String nextToken) {
+        this.limit = limit;
+        this.nextToken = nextToken;
+    }
+
+    /**
+     * TODO.
+     * @return todo.
+     */
+    public static ModelPagination firstPage() {
+        return nextToken(null);
+    }
+
+    /**
+     * TODO.
+     * @param nextToken todo.
+     * @return todo.
+     */
+    public static ModelPagination nextToken(@Nullable String nextToken) {
```

Comment:
Unused param

Reply:
It's a public API, the implementation needs to be updated after release to use the token. At least, I think that's whats being stated here, right, @drochetti?

Reply:
That's correct. It was a mistake, already fixed.